### DISCUSSION TITLE
Method to set up POST data for call to Coveo search service and subsequently call service with that data

### DISCUSSION THREAD

**Discussion by dale-vanzile** [coveo/cli#1004] [Q&A] [answered]
Hi - 
Newbie here. 

Need to find how to call the back end with my search term without navigating to a separate search page.  Have found in our app code the listener that triggers the navigation, and need to substitute a command or three to instead set up the data payload and call the back end search at `https://platform.cloud.coveo.com/rest/search/v2?organizationId=ourNiftyOrgIdString` to directly obtain the data without navigating to a search results page.

**Comment by louis-bompart** [comment, marked as answer]
Hey @dale-vanzile , this GH discussion 'forum' is mostly for CLI questions.
I invite you to join coveocommunity.slack.com and ask your question here.

Cheers!
